crates/stellar-registry-cli/src/commands/mod.rs

```diff
@@ -56,22 +60,30 @@ impl FromStr for Root {
 
 #[derive(Parser, Debug)]
 pub enum Cmd {
-    /// Publish Wasm to registry with package name and semantic version
-    Publish(Box<publish::Cmd>),
     /// Deploy a named contract from a published Wasm
     Deploy(Box<deploy::Cmd>),
+    /// Fetch a published Wasm
```

Comment:
This helpdoc just repeats the command name. What does it mean to fetch a published Wasm? Fetch it and do what with it? What can I do with it, once it is fetched?